api/model.go

```diff
@@ -812,59 +813,79 @@ func toVolumeResource(v *longhorn.Volume, ves []*longhorn.Engine, vrs []*longhor
 		}
 		backupStatus := e.Status.BackupStatus
 		if backupStatus != nil {
-			for id, status := range backupStatus {
+			ids := make([]string, 0, len(backupStatus))
```

Comment:
Suggest to extract the sort part as a common function in util  to reuse, ex: `GetSortedKeysFromMap`.

Reply:
Thank you, fixed!